environments/gcp/flyte-core/locals.tf

```diff
@@ -0,0 +1,19 @@
+locals {
+
+  #Change this on first use
+  application            = "flyte" #By default, this is used to build the FQDN for Flyte. See flyte-host notes
+  environment            = "gcp"
+  project_id             = "<your-GCP-projectID>"
+  dns-domain =           "uniondemo.run" #Change to your domain name 
+  region   =             "us-east1" #Change to your GCP region
+
+  # Change this only if you need to add more projects in the default installation
+  flyte_projects         = ["flytesnacks"]
+  flyte_domains          = ["development", "staging", "production"]
+  
+  #Don't change this
+  flyte_ksas             = ["default"] 
+  name_prefix            = "${local.application}-${local.environment}"
```

Comment:
If the user isn't supposed to change this, we could put these two locals into one of the other files where they are consumed.